custom/icds/serializers.py

```diff
@@ -19,7 +19,10 @@ class Meta(object):
 
     def to_representation(self, instance):
         ret = super(HostedCCZSerializer, self).to_representation(instance)
-        ret['app_name'] = self.context['app_names'][ret['app_id']]
+        if instance.build_doc:
+            ret['app_name'] = instance.build_doc['name']
+            custom_properties = instance.build_doc['profile'].get('custom_properties', {})
+            ret['app_version_tag'] = custom_properties.get('cc-app-version-tag')
```

Comment:
Not sure why we have `cc-app-version-tag` but using just `app_version` would make it clear its not something else.

Reply:
Not quite following, is it the naming of the `ret` key or the property that you find unclear? `cc-app-version-tag` us the same style as other app profile properties (`cc-superuser-enabled`, etc).

Reply:
naming in the ret.

Reply:
actually i see now that its actually a tag and not the version, so this is good.